Source/Krypton Components/Krypton.Ribbon/Controller/AppTabController.cs

```diff
@@ -369,9 +373,14 @@ public void KeyPress(Control c, KeyPressEventArgs e)
         /// <param name="ribbon">Reference to owning ribbon instance.</param>
         public void KeyTipSelect(KryptonRibbon ribbon)
         {
-            if ( ribbon.TabsArea is null )
+            if (ribbon is null)
```

Comment:
If truely nullabel at this point, then change the parameter type to be nullable

Reply:
@Smurf-IV 
Both files updated.

Reply:
@Smurf-IV 
Thank you!